terraform-plans/configs/charm-advanced-routing_main.tfvars

```diff
@@ -69,8 +69,11 @@ templates = {
     source      = "./templates/github/tox.ini.tftpl"
     destination = "tox.ini"
     vars = {
-      functest_type = "pytest"
-      unittest_type = "pytest"
+      functest_type         = "pytest"
+      unittest_type         = "pytest"
+      root_requirements_txt = "yes"
+      unit_requirements_txt = "yes"
+      func_requirements_txt = "yes"
```

Comment:
IMHO this goes over the line, introducing too much complexity to the template - three more low level variables. Can we find a better way please?

Reply:
As discussed in the meeting, we can use a single new variable: `is_python_project`.